src/schannel_cred.rs

```diff
@@ -248,7 +246,7 @@ impl Builder {
 
             match Identity::AcquireCredentialsHandleA(
```

Comment:
```suggestion
            match Identity::AcquireCredentialsHandleW(
```
`Identity::UNISP_NAME` is wide